### PR TITLE
Issue 15 fix, can now track latency of every pipelined request :)

### PR DESCRIPTION
**Pull request by thekemkid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ autocannon({
 Start autocannon against the given target, options:
 
 * `url`: the given target, mandatory
-* `connections`: the number of concurrent connections
+* `connections`: the number of concurrent connections, mandatory
```
Comment (davidmarkclements) [inline README.md]: how come mandatory, why no default?

Comment (mcollina) [inline README.md]: Do you think that 100 is a good value?

Comment (mcollina) [inline README.md]: It was my bad :/

Comment (thekemkid) [inline README.md]: You use 10 in the readme examples, I think using that is a good idea as it gives a clue to users about what to expect?

**Comment by thekemkid** [comment]
I'm going to close this as my new PR has this included within it.

**Pull request closed without merging**